### PR TITLE
[Merged by Bors] - feat: add absolutelyContinuous_zero_iff

### PR DESCRIPTION
**Pull request by LorenzoLuccioli**
Add lemma absolutelyContinuous_zero_iff to handle the case where a measure is absolutely continuous wrt 0.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean [hunk 1]
```diff
@@ -1628,6 +1628,12 @@ instance instIsRefl [MeasurableSpace α] : IsRefl (Measure α) (· ≪ ·) :=
 @[simp]
 protected lemma zero (μ : Measure α) : 0 ≪ μ := fun s _ ↦ by simp
 
+lemma eq_zero_of_ac_zero (h : μ ≪ 0) : μ = 0 := measure_univ_eq_zero.mp (h rfl)
+
```
Comment (LorenzoLuccioli) [inline Mathlib/MeasureTheory/Measure/MeasureSpace.lean:1632]:
```suggestion
```

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean [hunk 2]
```diff
@@ -1628,6 +1628,12 @@ instance instIsRefl [MeasurableSpace α] : IsRefl (Measure α) (· ≪ ·) :=
...
+@[simp]
+lemma ac_zero_iff : μ ≪ 0 ↔ μ = 0 :=
+  ⟨eq_zero_of_ac_zero, fun h ↦ h.symm ▸ AbsolutelyContinuous.zero _⟩
```
Comment (LorenzoLuccioli) [inline Mathlib/MeasureTheory/Measure/MeasureSpace.lean:1635]:
```suggestion
  ⟨measure_univ_eq_zero.mp (h rfl), fun h ↦ h.symm ▸ AbsolutelyContinuous.zero _⟩
```

File: Mathlib/MeasureTheory/Measure/MeasureSpace.lean [hunk 3]
```diff
@@ -1628,6 +1628,12 @@ instance instIsRefl [MeasurableSpace α] : IsRefl (Measure α) (· ≪ ·) :=
 @[simp]
 protected lemma zero (μ : Measure α) : 0 ≪ μ := fun s _ ↦ by simp
 
+lemma eq_zero_of_ac_zero (h : μ ≪ 0) : μ = 0 := measure_univ_eq_zero.mp (h rfl)
+
+@[simp]
+lemma ac_zero_iff : μ ≪ 0 ↔ μ = 0 :=
```
Comment (LorenzoLuccioli) [inline Mathlib/MeasureTheory/Measure/MeasureSpace.lean:1634]:
```suggestion
lemma absolutelyContinuous_zero_iff : μ ≪ 0 ↔ μ = 0 :=
```

**Comment by RemyDegenne** [comment]
Thanks!
bors r+

**Pull request closed without merging**